ui/apps/platform/src/Containers/Compliance/Entity/Cluster.jsx

```diff
@@ -46,7 +46,7 @@ function processData(data) {
     return data.cluster;
 }
 
-const ClusterPage = ({
+const Cluster = ({
```

Comment:
Should we remove the suffix? `Cluster` by itself seems more generic/ambiguous. 🤔 

Same question for the other alterations below.

Reply:
Yeah, let’s ask the team.
* On one side of coin, `ClusterPage` is more descriptive.
    Change to file names would imply changes to `import` statements in a few more files.
    Because of overlap with other containers, even more suffix seems more bettter: ComplianceClusterPage.jsx
    Although the extra effort does not in itself put me off, it might imply extra effort in rare case of backport.
* On other side of coin, this is consistent with Configuration Management.